lib/emittr/events.rb

```diff
@@ -59,6 +59,25 @@ def once_any(&block)
         once(:*, &block)
       end
 
+      def on_many_times(event, times, &block)
+        raise_no_block_error unless block_given?
+        raise ArgumentError, "#{times} must be an integer" unless times.is_a? Integer
+        raise ArgumentError, "#{times} can't be negative" if times < 0
+
+        callback = ::Emittr::Callback.new(&block)
+
+        off_block = proc do |args|
+          callback.call(*args)
+          block_to_send = callback.wrapper || callback.callback
+
+          off(event, &block_to_send) if times == 1
```

Comment:
What do you think about changing the order of this line and the next one to make it clearer when it should be `off`ed?